command/operator_debug.go

```diff
@@ -576,8 +613,13 @@ func (c *OperatorDebugCommand) collectNomad(dir string, client *api.Client) erro
 	vs, _, err := client.CSIVolumes().List(qo)
 	c.writeJSON(dir, "volumes.json", vs, err)
 
-	metrics, err := client.Operator().Metrics(qo)
-	c.writeJSON(dir, "metrics.json", metrics, err)
+	metricBytes, err := client.Operator().Metrics(qo)
+
+	if err != nil {
+		c.writeError(dir, "metrics.json", err)
+	} else {
+		c.writeBytes(dir, "metrics.json", metricBytes)
+	}
```

Comment:
nit: This can be compacted a bit 
```suggestion
	if metricBytes, err := client.Operator().Metrics(qo); err != nil {
		c.writeError(dir, "metrics.json", err)
	} else {
		c.writeBytes(dir, "metrics.json", metricBytes)
	}

```